### PR TITLE
New cache containers page

### PR DESCRIPTION
**Pull request by vblagoje**
New container cache page. Relays are prepared and added to API but not yet wired in UI. Will do so when we can inspect an actual relay object.

### CODE REVIEW

**Comment by vblagoje** [comment]
Here is the screenshot:

screen shot 2015-09-11 at 11 28 40 am

File: src/main/webapp/clusters-view/clusters-view.html [hunk 1]
```diff
@@ -1,46 +1,72 @@
...
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <tr ng-repeat="cluster in clusters track by $index">
+          <td><span class="label"
+                    ng-class="(cluster.isAvailable())?'label-success':'label-danger'">
+            {{cluster.isAvailable()?cluster.availability:'UNAVAILABLE'}}</span></td>
```
Comment (tristantarrant) [inline src/main/webapp/clusters-view/clusters-view.html]: We need to redefine this: availability is either AVAILABLE, DEGRADED or STOPPED. Next PR.

File: src/main/webapp/components/api/cluster-model.service.js [hunk 2]
```diff
@@ -63,6 +64,36 @@ angular.module('managementConsole.api')
               }.bind(this));
             };
 
+            Cluster.prototype.getEndpoints = function () {
+              var socketEndpoints = [];
+              var socketBindings = this.domain.getServerGroup().getSocketBindings();
+              var offset = this.domain.getServerGroup().getSocketPortBindingOffset();
+              var endpoints = this.domain.getProfile(this.profile).getEndpoints();
+              angular.forEach(endpoints, function(value, key, obj){
+                 var endpoint = value[key];
```
Comment (tristantarrant) [inline src/main/webapp/components/api/cluster-model.service.js]: I'd prefer to encapsulate all of the endpoint data in an Endpoint model class

Comment (vblagoje) [inline src/main/webapp/components/api/cluster-model.service.js]: Ok, I will do so.

Comment (vblagoje) [inline src/main/webapp/components/api/cluster-model.service.js]: Done @tristantarrant Have a look please

**Comment by tristantarrant** [comment]
Merged, thanks

**Pull request closed without merging**